cardano-cli/src/Cardano/CLI/Json/Friendly.hs

```diff
@@ -370,27 +370,34 @@ renderCertificate sbe = \case
   ConwayCertificate w cert ->
     conwayEraOnwardsConstraints w $
       case cert of
-        Ledger.RegDRepTxCert credential coin _TODO ->  -- TODO Conway: new StrictMaybe Anchor argument
+        Ledger.RegDRepTxCert credential coin mAnchor ->
           "Drep registration certificate" .= object
             [ "deposit" .= coin
             , "certificate" .= conwayToObject w credential
+            , "anchor" .= mAnchor
             ]
         Ledger.UnRegDRepTxCert credential coin ->
           "Drep unregistration certificate" .= object
             [ "refund" .= coin
             , "certificate" .= conwayToObject w credential
             ]
         Ledger.AuthCommitteeHotKeyTxCert coldCred hotCred
-            | Shelley.ScriptHashObj{} <- coldCred -> error "renderCertificate: TODO Conway era"
-            | Shelley.ScriptHashObj{} <- hotCred -> error "renderCertificate: TODO Conway era"
+            | Shelley.ScriptHashObj sh <- coldCred ->
+              "Cold committee authorization" .= object
+                [ "script hash" .= serialiseToRawBytesHexText (fromShelleyScriptHash sh) ]
+            | Shelley.ScriptHashObj sh <- hotCred ->
+              "Hot committee authorization" .= object
+                [ "script hash" .= serialiseToRawBytesHexText (fromShelleyScriptHash sh)]
             | Shelley.KeyHashObj (Shelley.KeyHash coldKey) <- coldCred
             , Shelley.KeyHashObj (Shelley.KeyHash hotKey) <- hotCred ->
-          "Constitutional committee member hot key registration" .= object
-            [ "cold key hash" .= String (textShow coldKey)
-            , "hot key hash" .= String (textShow hotKey)
-            ]
+              "Constitutional committee member hot key registration" .= object
+                [ "cold key hash" .= String (textShow coldKey)
```

Comment:
Showing this is going to make debugging difficult. We want to display the key hash as hex.